src/core/hle/service/cfg/cfg.h

```diff
@@ -53,13 +53,24 @@ struct EULAVersion {
     u8 major;
 };
 
+/// Access control flags for config blocks
+enum AccessFlag : u16 {
```

Comment:
```suggestion
enum class AccessFlag : u16 {
```
If you use [this](https://github.com/citra-emu/citra/blob/71582a72a46a2f9dc63365559ff10871f45b498e/src/common/common_funcs.h#L59) macro, it should allow for binary operators as well. Don't remember if it can also be used inside the enum, feel free to ignore this review if not.